simulations/basalt/interactive_simulation.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
Nice! Does it work out-of-the-box or are there additional instructions on how to get this up and running?